textalyze.js

```diff
@@ -9,12 +9,15 @@ let itemCounts = require('./lib/itemCounts');
 let readFileSync = require('./lib/readFileSync');
 let sanitize = require('./lib/sanitize');
 let stringToChars = require('./lib/stringToChars');
+let process = require('process');
+let file = process.argv[2];
```

Comment:
You'll want to check if the user supplied the appropriate number of arguments, e.g.,

```javascript
let fileName = process.argv[2];

if (fileName === undefined) {
  console.log('Error: No file given to analyze.');
  console.log('Usage:');
  console.log('  node textalyze.js <fileName>');
  process.exit();
}
```

You'll *probably* also want to check if the file exists and present the user with a nicer error message. You might do that elsewhere (I haven't looked), but it'd be natural to do it at the top so all the invalid argument errors are in the same place.

e.g.,

```javascript

let fileName = process.argv[2];

if (fileName === undefined) {
  console.log('Error: No file given to analyze.');
  console.log('Usage:');
  console.log('  node textalyze.js <fileName>');
  process.exit();
}

if (!fs.existsSync(fileName)) {
  console.log(`Error: No such file '${fileName}'`);
  console.log('Usage:');
  console.log('  node textalyze.js <fileName>');
  process.exit();
}
```

You might make some helper functions to remove duplication, e.g.,

```javascript
function printUsage() {
  console.log('Usage:');
  console.log('  node textalyze.js <fileName>');
}

function printError(errorMsg) {
  console.log(`Error: ${errorMsg}`);
}

function printErrorAndExit(errorMsg) {
  printError(errorMsg);
  printUsage();
  process.exit();
}

let fileName = process.argv[2];

if (fileName === undefined) {
  printErrorAndExit('Error: No file given to analyze.');
}

if (!fs.existsSync(fileName)) {
  printErrorAndExit(`No such file '${fileName}'`);
}
```